### PR TITLE
Fix zip.sh

### PR DESCRIPTION
**Pull request by mathieuhendey**

### CODE REVIEW

**Comment by zielinskikamil** [comment]
please remember to include *tsc* latter on when typescript will be ready

File: zip.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+npm run compile
+npm run copy-package
+cd build
+npm install --production
+rm ./*.json
```
Comment (danielchildsbjss) [inline zip.sh:5]: Probably don't need to do this? (Might cause some issues later)

**Review by danielchildsbjss — APPROVED** [review]
Looks good, probably don't need to remove *.json though.

File: zip.sh (hunk 1 above)
Comment (szymonrychu) [reply zip.sh:5]: +1- on Jenkins I'm always starting from scratch (removing workspace), so it's being handled on it's own.

**Pull request closed without merging**